winch/filetests/filetests/x64/call/params.wat

```diff
@@ -0,0 +1,99 @@
+;;! target = "x86_64"
```

Comment:
It might be good to have a test initially that exercises the spilling behavior (here we can't because `$add` is called in tail position). Perhaps call it multiple times, with some values on the stack that live "under" the call?

Reply:
Fixed! Thanks.